networking/external_dns_operator/nw-configuring-cluster-wide-egress-proxy.adoc

```diff
@@ -0,0 +1,10 @@
+:_content-type: ASSEMBLY
+[id="nw-external-dns-operator-egress-proxy"]
+= Configuring cluster-wide proxy
+:context: external-dns-operator
+
+toc::[]
```

Comment:
Yes, looking into it.

Reply:
Figured out the reason why TOC was not showing it's because `include::_attributes/common-attributes.adoc[]` was missing. After adding `include::_attributes/common-attributes.adoc[]` ; TOC is showing

Reply:
Addressed